.drone.yml

```diff
@@ -149,11 +149,116 @@ services:
       - name: dockersock
         path: /var/run
 
+volumes:
+  - name: dockersock
+    temp: {}
+---
+kind: pipeline
+type: kubernetes
+name: deploy docs
+
+trigger:
+  event:
+  - push
+  branch:
+  - master
+
+clone:
+  disable: true
```

Comment:
This is necessary because the web working copy expects to have a specific relative location to the gravity working copy:

https://github.com/gravitational/web/blob/956cfafe7c636f0ea6418f48461ca5d917347593/deploy/gravity-docs.yaml#L9